### ISSUE TITLE
CVE-2021-27290

### ISSUE THREAD

**Issue by taisph** [JeffreyWay/laravel-mix#2952]
-   Laravel Mix Version: 5.0.9 (`npm list --depth=0`)
-   Node Version (`node -v`): 14.15.5
-   NPM Version (`npm -v`): 6.14.11
-   OS: Debian

### Description:

High severity CVE-2021-27290 in ssri >= 7.0.0, < 8.0.1

```
=> Found "ssri@7.1.0"
info Reasons this module exists
   - "laravel-mix#terser-webpack-plugin#cacache" depends on it
   - Hoisted from "laravel-mix#terser-webpack-plugin#cacache#ssri"
=> Found "webpack#ssri@6.0.2"
info Reasons this module exists
   - "laravel-mix#webpack#terser-webpack-plugin#cacache" depends on it
   - Hoisted from "laravel-mix#webpack#terser-webpack-plugin#cacache#ssri"

=> Found "cacache@13.0.1"
info Reasons this module exists
   - "laravel-mix#terser-webpack-plugin" depends on it
   - Hoisted from "laravel-mix#terser-webpack-plugin#cacache"
=> Found "webpack#cacache@12.0.4"
info Reasons this module exists
   - "laravel-mix#webpack#terser-webpack-plugin" depends on it
   - Hoisted from "laravel-mix#webpack#terser-webpack-plugin#cacache"

=> Found "terser-webpack-plugin@2.3.8"
info Reasons this module exists
   - "laravel-mix" depends on it
   - Hoisted from "laravel-mix#terser-webpack-plugin"
=> Found "webpack#terser-webpack-plugin@1.4.5"
info This module exists because "laravel-mix#webpack" depends on it.
```

**Comment by thecrypticace** [comment]
Does a `npm update` / `yarn upgrade` fix this for you? ssri is a transitive dependency and there's nothing _we_ can do to fix that but it looks like they released 6.0.2 a while back to address the CVE.

**Comment by taisph** [comment]
Both ssri 7.x and 6.x have been fixed and the advisory has been adjusted to reflect this, so upgrading laravel-mix now fixes this issue.
